src/SFA.DAS.EmployerIncentives/Infrastructure/Api/RestApiClient.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.WebUtilities;
+using SFA.DAS.EmployerIncentives.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Infrastructure.Api
+{
+    public class RestApiClient : IRestApiClient
```

Comment:
There's a RestHttpClient in the SFA.DAS.Http Nuget package, can that be inherited from instead?

Reply:
We are not using the Http Package as it's got references to Structure map and the httpClient. Also because I didn't think it had been reviewed. I added the updates to this branch to make core 3.1. Would you kindly take another look thx